docs/api/flink/Function.md

```diff
@@ -616,9 +616,8 @@ SELECT ST_Buffer(ST_GeomFromWKT('POINT(0 0)'), 10, false, 'quad_segs=2')
 ```
 
 Output:
-
-<img alt="Point buffer with 8 quadrant segments" src="../../../image/point-buffer-quad-8.png" width="100" height=""/>
-<img alt="Point buffer with 2 quadrant segments" src="../../../image/point-buffer-quad-2.png" width="100" height=""/>
+![Point buffer with 8 quadrant segments](../../image/point-buffer-quad-8.png){: width="100px"}
```

Comment:
When viewing the docs on GitHub this type of syntax does not work with `{: width="100px"}` being displayed as is.

![Screenshot from 2024-04-18 02-07-06](https://github.com/apache/sedona/assets/418747/862897f6-b421-4071-91cf-dde2e6c62d2c)

Reply:
The syntax "{: width="100px"}" is not supported by github. Since there are not that many pictures, what we could do is resize all pictures, and get rid of this. And possibly have a linter? rule on top  of it.

Reply:
Best practice for web development is to serve images with the correct dimensions and avoid scaling down the images with HTML so that you don't waste users' data and hurt page performance.

https://web.dev/articles/serve-images-with-correct-dimensions

Reply:
I issued a new pull request: https://github.com/apache/sedona/pull/1359, hope it helps!